cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -30,100 +30,122 @@
     <div class="block">
         {{ social_media.render() }}
     </div>
-    <div class="block">
-        <div class="expandable-prototypes">
-            {% include 'molecules/expandable.html' with context %}
-
-
-            {% do global_dict.update(
-                {
-                    'expandable': {
-                        'label': 'Bordered Expandable',
-                        'paragraph': lipsum(1, false, 10, 225),
-                        'is_bordered': true,
-                        'is_midtone': true,
-                        'is_expanded': false
-                    }
-
-                })
-            %}
-            {% set value = global_dict.expandable %}
-            {% include 'molecules/expandable.html' with context %}
-
-            {% do global_dict.update(
-                {
-                    'expandable': {
-                        'label': 'Midtone Expandable',
-                        'paragraph': lipsum(1, false, 10, 225),
-                        'is_bordered': true,
-                        'is_midtone': true,
-                        'is_expanded': false
-                    }
-
-                })
-            %}
-            {% set value = global_dict.expandable %}
-            {% include 'molecules/expandable.html' with context %}
-
-             {% do global_dict.update(
-                 {
-                    'expandable': {
-                        'label': 'Midtone, Bordered, Expandable - Expanded',
-                        'paragraph': lipsum(1, false, 10, 225),
-                        'is_bordered': true,
-                        'is_midtone': true,
-                        'is_expanded': true
-                    }
-                 })
-             %}
-            {% set value = global_dict.expandable %}
-            {% include 'molecules/expandable.html' with context %}
-            {% set value = None if prototype %}
+
+    {% for block in page.header -%}
+        <div class="block">
```

Comment:
You're right, @anselmbradford -- areas 1 and 2 are schematic indicators to help everyone understand hierarchy and organization of organisms within the page type. They do not need to be visually apparent to end users.